person.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'nameable'
+
+class Person
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
+
+  def initialize(age, name, parent_permission)
+    @id = Random.rand(1..999_999_999)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def can_use_services?
+    return unless of_age || @parent_permission
+
+    true
+  end
+
+  def correct_name
+    @name
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+end
```

Comment:
- Please implement a method to add rentals, the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Please fix this 👍 👍

Reply:
Method add_rentals added.